mne/time_frequency/psd.py

```diff
@@ -1,12 +1,13 @@
 # Authors : Alexandre Gramfort, alexandre.gramfort@telecom-paristech.fr (2011)
 #           Denis A. Engemann <denis.engemann@gmail.com>
+#           Mikolaj Magnuski <mmagnuski@swps.edu.pl>
```

Comment:
I'm not sure if I should add myself here :) If so - there are probably other people I should add here.